src/commands/create-fixture.ts

```diff
@@ -1,46 +1,90 @@
-import { getPRInfo, queryPRInfo } from "../pr-info";
+import * as assert from "assert";
 import * as computeActions from "../compute-pr-actions";
-import { render } from "prettyjson";
-import { writeFileSync, mkdirSync, existsSync } from "fs";
+import { deriveStateForPR, queryPRInfo, PrInfo } from "../pr-info";
+import { writeFileSync, mkdirSync, existsSync, readFileSync } from "fs";
 import { join } from "path";
 import { getOwnersOfPackages } from "../util/getOwnersOfPackages";
+import { executePrActions } from "../execute-pr-actions";
+import { getMonthlyDownloadCount } from "../util/npm";
 
-
-async function main() {
-  const num = +process.argv[2];
-  
-  const fixturePath = join("src", "_tests", "fixtures", num + "")
-
-  const response = await queryPRInfo(num);
-  const derivedInfo = await getPRInfo(num);
-
+export default async function main(prNumber: number, overwriteInfo: boolean) {
+  const fixturePath = join("src", "_tests", "fixtures", prNumber + "")
   if (!existsSync(fixturePath)) mkdirSync(fixturePath)
 
   const jsonFixturePath = join(fixturePath, "_response.json")
-  writeFileSync(jsonFixturePath, JSON.stringify(response, null, "  ")) 
+  let response
+  if (overwriteInfo || !existsSync(jsonFixturePath)) {
```

Comment:
Create fixture can now be updated with or without re-querying the input info (owners, downloads, PR state). Useful for updating existing output fixtures without modifying the inputs (e.g. when result.json and friends change due to logic updates)